dgbpy/bokeh_apps/trainui/main.py

```diff
@@ -345,6 +345,18 @@ def stopTrainingCB( stoptrainingcheckbox ):
       stoptraining = getUiParams['stopaftercurrentepoch']
     return None
 
+  saveonabort = True
+
+  def saveOnAbortCB( saveonabortcheckbox ):
+    nonlocal platformfld
+    nonlocal saveonabort
+    selplatform = platformfld.value
+    if selplatform != uisklearn.getPlatformNm():
```

Comment:
Do not write if ! else block. If you have two implementations (for if true and else), make them in the logical order